### ISSUE TITLE
[Bug]: Sites section is empty, but herd links are not

### ISSUE THREAD

**Issue by Focusnik** [beyondcode/herd-community#318]
### Operating system version

macOS Monterey 12.4

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.4.0 (Build: 20)

### PHP Version

PHP 8.3.2 (cli) (built: Feb  1 2024 12:56:44) (NTS)

### Bug description

I've removed valet and make fresh installation for Herd. Clear all sites and create new one with "herd link". Sites section in settings is empty, but "herd links" return a list.

### Steps to reproduce

1. Remove valet
2. Fresh install of Herd
3. Clear all sites, if exist
4. Add a new link with "herd link ..."
4. Check sites section and "herd links"

<img width="905" alt="Снимок экрана 2024-02-06 в 14 04 22" src="https://github.com/beyondcode/herd-community/assets/3830450/302b2d59-af59-485b-9b45-24a54fd86b78">

<img width="906" alt="Снимок экрана 2024-02-06 в 14 04 13" src="https://github.com/beyondcode/herd-community/assets/3830450/a310e2e1-b063-421f-b910-216381860934">

<img width="1268" alt="Снимок экрана 2024-02-06 в 14 04 35" src="https://github.com/beyondcode/herd-community/assets/3830450/395e51f0-f647-4d48-8a9a-f10cba88a98f">

**Comment by sschlein** [comment]
What happens if you park the whole folder if your sites with Herd instead of using herd link on every site?

**Comment by Focusnik** [comment]
nothing, sites section still empty. But "herd links" won't changed too

**Comment by hope22** [comment]
Same for me. All sites gone.

**Comment by sschlein** [comment]
Could you please send me your output of `herd links` to support@beyondco.de ? As text and/or screenshots would be great.

I assume that something is breaking the response and that's hard to get without having the real data.

Do these sites work?

**Comment by hope22** [comment]
```
❯ herd links
+------+-----+-----+------+-------------+
| Site | SSL | URL | Path | PHP Version |
+------+-----+-----+------+-------------+

~
❯
```

All missing for me since update.

**Comment by Focusnik** [comment]
Sure, all sent.

Sites works properly, without any problems, but not displayed in sites section.

**Comment by sschlein** [comment]
@Focusnik we'll look into that via email support, it's good that the sites still work and seems to be a display issue that we need to track down and post a solution here after we find it so that it's possible to google it

@hope22 That seems to be a different problem, can you please send me your herd.conf and nginx.conf to support@beyondco.de ? please also let me know if you followed all the uninstall steps at https://herd.laravel.com/docs/1/troubleshooting/uninstalling#uninstalling-herd

There might also be a nginx process that go mad somehow and that you need to kill. if you don't find the nginx process in your activity monitor, please make sure that you show all and not only user related processes. You can do that in the main menu of the activity monitor. Unfortunately, I can only provide a German screenshot, but it should be at the same localtion

![CleanShot 2024-02-06 at 11 51 02](https://github.com/beyondcode/herd-community/assets/2911113/ffd0fb88-9e2f-47b8-b2d3-814b401032f4)

**Comment by hope22** [comment]
Thanks @sschlein im currently talking to support. Managed to sort the zombie process, trying to solve the rest now.

**Comment by michaelklopf** [comment]
FYI the site page is broken for me and my colleague too

**Comment by mpociot** [comment]
@michaelklopf but the sites work in general? So it's mostly a UI thing in the Herd App?

**Comment by michaelklopf** [comment]
@mpociot yes, only a UI thing in the Herd App, sites can be opened

**Comment by sschlein** [comment]
So what do have all you guys in common? 🤔

**Comment by carlosten** [comment]
Same problem here. I also have macOS Monterey. Maybe a Monterey issue?

**Comment by abarkhuysen** [comment]
I have the following to add:
All my sites work as expected but the site list is empty in the Herd UI. 
Running Herd 1.4.0 (Build: 20) on Mac Mini M2 Pro and Sonoma 14.2.1.

1. The herd links command displays an empty list
```
> herd links
+------+-----+-----+------+-------------+
| Site | SSL | URL | Path | PHP Version |
+------+-----+-----+------+-------------+
```
2. The herd parked command displays the complete list of my site as expected
```
> herd parked
+------------------------------+-----+------------------------------------------+---------------------------------------------------+-----+---------+
| Site                         | SSL | URL                                      | Path                                              | PHP | Node    |
+------------------------------+-----+------------------------------------------+---------------------------------------------------+-----+---------+
| .idea                        |     | http://.idea.test                        | /Users/arthur/Templates/.idea                     | 8.1 | v21.5.0 |
```
3. herd secured also works as expected

Not sure if any of this will be helpful :)

**Comment by anotherfrontendguy** [comment]
> Same problem here. I also have macOS Monterey. Maybe a Monterey issue?

I'm afraid not, it's happening for me on macOS Sonoma 14.3 as well.

**Comment by sschlein** [comment]
> I'm afraid not, it's happening for me on macOS Sonoma 14.3 as well.

At least we don't need to setup Monterey 🙈 

Thanks for the update/info!

**Comment by mpociot** [comment]
Which shell do you use?

**Comment by anotherfrontendguy** [comment]
fish

**Comment by mkwsra** [comment]
Thanks guys, and yes me and my team we have the same issue and all of us we use `fish`

**Comment by abarkhuysen** [comment]
> Which shell do you use?

I use -zsh on Sonoma - default install

**Comment by gregorysouzasilva** [comment]
I was having the same issue of not showing the sites. However even herd links was failing:
Error finding executable PHP. Quitting for safety.
Provided output from find-usable-php.php:
/Users/gregorysouzasilva/Library/Application Support/Herd/bin/php83

Reselecting the php version on the menu solved for me.

**Comment by neilgilmour** [comment]
Same here after upgrading and going pro - `herd parked` and `herd links` both show sites, but only the old sites set up through valet (ie symlinked in /Users/neil/Library/Application Support/Herd/config/valet/Sites/ ) work in the browser. Also can't install node with Herd.

M1 max, Sonoma 14.4, fish shell

**Comment by mpociot** [comment]
Yeah it's related to version 1.4.0 (not Pro). 
We need a sourced shell in order to read Node information per project, which currently does not work in combination with fish. I'm already working on a fix

**Comment by michael-grunewalder** [comment]
> Yeah it's related to version 1.4.0 (not Pro). We need a sourced shell in order to read Node information per project, which currently does not work in combination with fish. I'm already working on a fix

In case it helps with debugging:
I used the nu shell, reverted to zsh, rebooted but the list stays empty.
Everything else seems to work.

Thanks

**Comment by susanBuck** [comment]
Having the same issue. Sites all work, but not displaying in GUI or `herd links` output.

M2
Ventura v13.6.4 
I use zsh

```
$ herd links
+------+-----+-----+------+-------------+
| Site | SSL | URL | Path | PHP Version |
+------+-----+-----+------+-------------+
```

<img width="910" alt="image" src="https://github.com/beyondcode/herd-community/assets/237711/06834aa5-b976-4266-a6b3-5dd47f4e3c6f">

**Comment by sefaiyi** [comment]
Hi everyone

after installing the latest version, the site list disappeared in gui. Is there anyone who can solve this problem?

- by the way, the sites are still working, but the list in the gui is empty.

I guess we have to wait for a new version. -> to get the list back

![image](https://github.com/beyondcode/herd-community/assets/4063985/7c965e7a-5ee5-4c40-8d25-f95010d06f6f)

**Comment by mpociot** [comment]
This is already fixed in the next update which will be released today.

**Comment by sefaiyi** [comment]
thank @mpociot great news

**Comment by mpociot** [comment]
We just released Herd 1.4.1 which fixes this

**Comment by sefaiyi** [comment]
it's work rocksssssssss 👍
